dist/Data-Dumper/t/dumper.t

```diff
@@ -93,26 +103,36 @@ sub convert_to_native($) {
     return join "", grep { defined } @output;
 }
 
+sub add_my_to_dump {
```

Comment:
this helper seems to be used in more than one location, maybe it could be shared

Reply:
I agree but I'm only adding it here. I'd prefer to take it out in a follow up commit.